src/storage/src/mem_table.rs

```diff
@@ -81,18 +85,25 @@ impl MemTable {
     /// write methods
     pub fn insert(&mut self, pk: Bytes, value: Bytes) -> Result<()> {
         if !self.is_consistent_op {
-            self.buffer.insert(pk, KeyOp::Insert(value));
+            let key_len = pk.len();
+            self.kv_size.add(&pk, &value);
+            let origin_value = self.buffer.insert(pk, KeyOp::Insert(value));
+            self.calculate_origin_size(origin_value, key_len);
```

Comment:
Can we drive a EstimateSize on KeyOp and estimate it? It can be more accurate.
```
#[derive(Clone, Debug, EstimateSize)]
pub enum KeyOp {
    Insert(Bytes),
    Delete(Bytes),
    /// (old_value, new_value)
    Update((Bytes, Bytes)),
}
```